packages/@aws-cdk/aws-vpcv2-alpha/lib/util.ts

```diff
@@ -0,0 +1,434 @@
+/*eslint no-bitwise: ["error", { "allow": ["~", "|", "<<", "&"] }] */
+
+import { ISubnet, SubnetType } from 'aws-cdk-lib/aws-ec2';
+
+/**
+ * Turn an arbitrary string into one that can be used as a CloudFormation identifier by stripping special characters
+ *
+ * (At the moment, no efforts are taken to prevent collisions, but we can add that later when it becomes necessary).
+ */
+export function slugify(x: string): string {
+  return x.replace(/[^a-zA-Z0-9]/g, '');
+}
+
+/**
+ * The default names for every subnet type
+ */
+export function defaultSubnetName(type: SubnetType) {
+  switch (type) {
+    case SubnetType.PUBLIC: return 'Public';
+    case SubnetType.PRIVATE_WITH_NAT:
+    case SubnetType.PRIVATE_WITH_EGRESS:
+    case SubnetType.PRIVATE:
+      return 'Private';
+    case SubnetType.PRIVATE_ISOLATED:
+    case SubnetType.ISOLATED:
+      return 'Isolated';
+  }
+}
+
+/**
+ * Return a subnet name from its construct ID
+ *
+ * All subnet names look like NAME <> "Subnet" <> INDEX
+ */
+export function subnetGroupNameFromConstructId(subnet: ISubnet) {
+  return subnet.node.id.replace(/Subnet\d+$/, '');
+}
+
+/**
+ * Make the subnet construct ID from a name and number
+ */
+export function subnetId(name: string, i: number) {
+  return `${name}Subnet${i + 1}`;
+}
+
+/**
+ * Generate the list of numbers of [0..n)
+ */
+export function range(n: number): number[] {
```

Comment:
Where is this function used?

Reply:
this is not in use currently in new construct, I see it was being used in the `importSubnetGroup` earlier, since we haven't worked on imports yet, will remove it from this class